### ISSUE TITLE
Array conv_23/Conv_1/BatchNorm/FusedBatchNorm, which is an input to the Conv operator producing the output array conv_3//0/Conv/Relu6, is lacking min/max data, which is necessary for quantization. 

### ISSUE THREAD

**Issue by sxsxsx** [tensorflow/tensorflow#27952]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):NO
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):win10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):conda install tensorflow-gpu==1.13
- TensorFlow version (use command below):1.13
- Python version: 3.6
- Bazel version (if compiling from source):NO
- GCC/Compiler version (if compiling from source):NO
- CUDA/cuDNN version: CUDA9.0 cuDNN7.1
- GPU model and memory: p40

**Describe the current behavior**
I quantized training mobilefacenet,   and want to conver pb file to tflite file but failed

toco --output_file=face_freeze.tflite --graph_def_file=face_freeze.pb --input_arrays=x_i
nput  --output_arrays=BatchNorm/Reshape_1 --output_format=TFLITE --inference_typ
e=QUANTIZED_UINT8 --mean_values=127  --std_dev_values=128

**Code to reproduce the issue**
`def model(is_training , x, y, is_quantize ):

    with slim.arg_scope([slim.conv2d, slim.fully_connected],
                        activation_fn = tf.nn.relu6,
                        weights_initializer=slim.variance_scaling_initializer(factor=3.0, mode='FAN_IN', uniform=True),					
                        biases_initializer = tf.constant_initializer(0.0), 						
                        normalizer_fn=slim.batch_norm,    
                        normalizer_params={'is_training': is_training, 'epsilon':0.001, 'scale': True, 'decay':0.9, 'updates_collections': tf.GraphKeys.UPDATE_OPS}):

        print('x',x.get_shape())        
        # conv_1        
        conv_1 = slim.conv2d(x, 64, [3, 3], stride=2, padding='SAME',  scope='conv_1')
        print('conv_1',conv_1.get_shape())

        # conv_2
        conv_2 = Residual(conv_1, num_block=2, num_out=64, _kernel=3,  _stride=1, pad='SAME', num_group=64, name='conv_2')         
        print('conv_2',conv_2.get_shape())

       # conv_23
        conv_23 = DResidual(conv_2, num_out=128, _kernel=3 , _stride=2, pad='SAME', num_group=128, name='conv_23')
        print('conv_23',conv_23.get_shape())

       # conv_3
        conv_3 = Residual(conv_23, num_block=6, num_out=128, _kernel=3, _stride=1, pad='SAME', num_group=128, name='conv_3')
        print('conv_3',conv_3.get_shape())
        .....
        .....
		
        #softmax loss
            cross_entropy__ = tf.nn.sparse_softmax_cross_entropy_with_logits(
                labels=y, logits=fc7, name='cross_entropy_per_example')
        cross_entropy = tf.reduce_mean(cross_entropy__, name='cross_entropy')

        all_variable_decay = _all_variables_decapy(weight_decay,1.0)		
        total_loss = cross_entropy + all_variable_decay 
			
        tf.contrib.quantize.create_training_graph()`

I use tf.contrib.quantize.create_eval_graph() to create eval graph pb file

**Other info / logs**
2019-04-18 19:21:14.747016: F tensorflow/lite/toco/tooling_util.cc:1702] Array c
onv_23/Conv_1/BatchNorm/FusedBatchNorm, which is an input to the Conv operator p
roducing the output array conv_3//0/Conv/Relu6, is lacking min/max data, which i
s necessary for quantization. If accuracy matters, either target a non-quantized
 output format, or run quantized training with your model from a floating point
checkpoint to change the input graph to contain min/max information. If you don'
t care about accuracy, you can pass --default_ranges_min= and --default_ranges_m
ax= for easy experimentation.

Looking forward to receive your reply,  thanks very much

**Comment by avishek15** [comment]
I am facing a similar issue with Cifrarnet as well. I have trained the cifarnet model with quantization awareness. The model even converts into frozen model. During the conversion with tflite_converter, I get the following error:

```
2019-05-05 08:56:39.657151: F tensorflow/lite/toco/tooling_util.cc:1702] Array CifarNet/norm1, which is an input to the Conv operator producing the output array CifarNet/conv2/Relu, is lacking min/max data, which is necessary for quantization. 
If accuracy matters, either target a non-quantized output format, or run quantized training with your model from a floating point checkpoint to change the input graph to contain min/max information. 
If you don't care about accuracy, you can pass --default_ranges_min= and --default_ranges_max= for easy experimentation.
Aborted (core dumped)
```
Are there certain ops for which fake quant node needs to be manually added? Any insight into this will be of great help.

Thanks.

**Comment by sushreebarsa** [comment]
@sxsxsx Is this still an issue?
We see that you are using an older version of TF(1.x) which is not actively supported.We recommend you to kindly upgrade to TF v2.4 or later and let us know the outcome? Please refer to this [thread ](https://stackoverflow.com/questions/63681168/batch-normalization-quantize-tensorflow-1-x-does-not-have-minmax-information) and let us know if it helps?
Thanks!
